site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -604,15 +604,15 @@ Be sure to filter for malicious web pages. For example, the following patterns a
 
 {% Compare 'worse' %}
 ```js
-var data = document.getElementById("json-data")
+const data = document.getElementById("json-data")
 // WARNING! Might be evaluating an evil script!
-var parsed = eval("(" + data + ")")
+const parsed = eval("(" + data + ")")
```

Comment:
Out of scope for this PR, but we should remove references to eval() due to Manifest V3's remotely hosted code restrictions. (See #1226)

Reply:
Well, I've added a line here.

> Be sure to filter for malicious web pages. For example, the following patterns are dangerous, and disallowed in MV3: